src/main/scala/ch/openolitor/buchhaltung/rechnungsexport/iso20022/Pain008_001_02_Export.scala

```diff
@@ -69,7 +69,7 @@ class Pain008_001_02_Export extends LazyLogging {
   }
 
   private def getGroupHeaderSDD(rechnungen: List[Rechnung], kontoDatenProjekt: KontoDaten, nbTransactions: String): GroupHeaderSDD = {
-    val MsgId = kontoDatenProjekt.iban + getDateTime().toString()
+    val MsgId = kontoDatenProjekt.iban.get + randomAlphaNumericString(34 - kontoDatenProjekt.iban.get.length)
```

Comment:
Is there really a random string needed?

Reply:
This is the id for the message. It is good to have the reference of the iban at the beginning to being able to track it in case it is needed. We can create multiple files at the same time from the same initiative so it is good to have some other component to identify them. That is why of the random generated part. The time could also be used but the field is max 35 chars and the iban is max of 34 chars.

Reply:
fine for me.